### PR TITLE
No SDPover SCP

### PR DESCRIPTION
**Pull request by rmosimann**
Hi there, 
Please review this pull request, DON'T MERGE. 
The Server has to be changed too, to make it work. 

It is a undo of uploading the SDP over SCP.

### CODE REVIEW

File: resources/docker-build_p2p/port_connect.sh
Comment (martinfrancois) [inline resources/docker-build_p2p/port_connect.sh]: is it necessary to put -o Strict... here again? it is already present in ssh and will be featured here again, as it is included with $ssh

File: resources/docker-build_p2p/port_share.sh
Comment (martinfrancois) [inline resources/docker-build_p2p/port_share.sh]: see above comment on -o Strict...

File: resources/docker-build_p2p/ssh.rc
Comment (martinfrancois) [inline resources/docker-build_p2p/ssh.rc]: don't forget to change this back

**Review by martinfrancois — COMMENTED** [review]
Looks good all in all. How do we now transfer the Dump if not over SCP?
Since now Mr. Standtke might be trying out our RSCC, we need to be more careful with changes on the server. We might have to think about how we want to do this exactly so nothing gets broken.

**Comment by rmosimann** [comment]
Yes, we have to discuss how we handle this, when our customer is on the Master-Branch.
We don't push the SDP per file, we exchange this in the TCP connection between the Clients. this is what the RSCCCFP is for.

**Comment by pwigger** [comment]
Can you guide me through? we exchange the SDP over TCP so far. Secure Copy (SCP) as protocol over TCP? but it doesnt work? or we dont use sdp now and you want to add it for security reasons?

**Comment by rmosimann** [comment]
@pwigger It was all prepared to exchange the stuff over SCP. But we do not. So this has to be removed. this is what this PR does. But there is a correlation to the Server. so this has to be done synchronized. 
That is why i'm still waiting for approval. should be done before our customer goes online with his server.

**Comment by martinfrancois** [comment]
@jpduloch did you discuss yesterday with Mr. Standtke how to manage that?

**Review by martinfrancois — APPROVED** [review]
looks good to me!

**Comment by rmosimann** [comment]
I changed the server to the latest dockerimage (new Container: "p2platest". old container "P2P" is still available.)
Merging this to implement the changes on the client-side. 
All currently open branches need to merge develop to communicate with the server.

**Pull request merged**
